### DISCUSSION TITLE
Not seeing all stations for a specific office and gridpoint set?

### DISCUSSION THREAD

**Discussion by lemonprogis** [weather-gov/api#669] [Q&A] [answered]
I am getting endpoints from my location:

https://api.weather.gov/points/35.0202803,-92.4754276

From there i have the forecast and stations endpoints. 

https://api.weather.gov/gridpoints/LZK/75,85/forecast
https://api.weather.gov/gridpoints/LZK/75,85/stations

Forecasts look good (verifying from weather.gov/lzk and clicking on the map of the same location)

When i retrieve the stations, i am not seeing KCXW in my list even though my lat/lng is close to it. I just see the closest to me is adams field (KLIT). 

Any help would be most appreciated, thanks!

**Comment by dwhitemv25** [comment]
The station list is curated by the WFO, and they may feel that KCXW is not representative of the area (or a higher quality station is nearby). 

Checking some other sources, it looks like KCXW is an unattended AWOS station with an older/simpler sensor package (METAR code AO1).  That might be why they aren't listing it.

**Reply by lemonprogis** [reply]
Ok, that's interesting and thank you for the reply! 

The station in question is about 20 miles from the closest one in the list(Adams field). When I visit weather.gov/lzk and click on conway, it shows data from KCXW for current observations. I was hoping to achieve the same result using the API. I guess the next step for me would be to aggregate the grid point data to be the 'current observation' for a location to achieve similar results?

**Reply by sullynole** [reply, marked as answer]
It also doesn't show up under "More Local Wx" from the forecast page for the zone it is in either: https://forecast.weather.gov/obslocal.php?warnzone=ARZ032&local_place=Conway%20AR&zoneid=CDT&offset=18000.  The local office should have the ability to add this office there.  Might reach out the webmasters at WFO Little Rock: https://www.weather.gov/lzk/email.htm

**Reply by lemonprogis** [reply]
Yep, I noticed that and thought it should have been there. Appreciate the information, I'll reach out to my WFO and see.
